src/Hystrix.Dotnet/HystrixOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Hystrix.Dotnet
+{
+    public class HystrixOptions
+    {
+        public HystrixOptions()
+        {
+            ConfigurationServiceImplementation = "HystrixLocalConfigurationService";
```

Comment:
I think all of these default values can be initialised directly in the properties, e.g.:

```
public string ConfigurationServiceImplementation { get; set; } = "HystrixLocalConfigurationService";
```

and it would contribute to have less boilerplate, IMO